src/Shared/SFA.DAS.ProviderPayments.Calc.Shared.IntegrationTests/Attributes/SetupDasAccountsAttribute.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Linq;
+using AutoFixture;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using SFA.DAS.ProviderPayments.Calc.Shared.Infrastructure.Data.Entities;
+using SFA.DAS.ProviderPayments.Calc.Shared.IntegrationTests.Helpers;
+using SFA.DAS.ProviderPayments.Calc.Shared.IntegrationTests.SpecimenBuilders;
+
+namespace SFA.DAS.ProviderPayments.Calc.Shared.IntegrationTests.Attributes
+{
+    public class SetupDasAccountsAttribute : TestActionAttribute
+    {
+        public override ActionTargets Targets { get; } = ActionTargets.Suite;
+
+        public override void BeforeTest(ITest test)
+        {
+            DasAccountDataHelper.Truncate();
+
+            var fixture = new Fixture();
+            fixture.Customizations.Add(new UnknownAccountIdBuilder());
+
+            var dasAccounts = fixture.Build<DasAccountEntity>()
```

Comment:
I'm not an expect on AutoFixture, but how do we stop the AccountId from being duplicated here? the Specimen builder doesn't seem to handle this case (as far as I can see)

Reply:
compare these two:
https://github.com/SkillsFundingAgency/das-providerpayments/pull/281/files#diff-43b2238d6c9d550cb915c4bb3f0524cd
and
https://github.com/SkillsFundingAgency/das-providerpayments/pull/281/files#diff-b7c6e45e829276f6812479f1327e9b1e

Each pulls random values from a different range than the other.